src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/SearchInputDetails.scala

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+
+object SearchInputDetails extends PerformanceTestsConfig {
+
+  private val url: String = config.getString("caseDataUrl") + "/" + config.getString("searchInputDetails")
+  println("Retrieving searchInputDetails URL : " + url)
+
+  def httpRequest() = {
+    val s2sToken = CcdTokenGenerator.generateGatewayS2SToken()
+    val userToken = CcdTokenGenerator.generateWebUserToken()
+    http("TX07_CCD_SearchInputDetails_searchInputDetails")
```

Comment:
can we call this `CCD_SearchInputDetails_searchInputDetails`. what is TX07 used for?

Reply:
TX01, TX02.....TXxx all are transactions name.